wordpress-mobile/WordPress-iOS
# Reader: Update `ReaderTagCell` to new UI

Fixes #23184 

## Description

Updates the Reader tag cell to the new UI: p1715247952499939/1715197207.367749-slack-C05N140C8H5

## Testing

To test:
- Launch Jetpack and login
- Enable the "Reader Tags Feed" feature flag
- Navigate to the "Your Tags" feed
- After the feed loads, 🔎 **verify** the UI is updated to the UI referenced above 

## Regression Notes
1. Potential unintended areas of impact
Should be none.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A
PR submission checklist:


Testing checklist:
